conda_forge_tick/auto_tick.xsh

```diff
@@ -121,6 +121,43 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
     return migrate_return, pr_json
 
 
+def add_rebuild(migrators, gx):
+    """Adds rebuild migrators.
+
+    Parameters
+    ----------
+    migrators : list of Migrator
+        The list of migrators to run.
+
+    """
+
+    pygraph = copy.deepcopy(gx)
+    r_graph = copy.deepcopy(gx)
+    compiler_graph = copy.deepcopy(gx)
+    openblas_graph = copy.deepcopy(gx)
+    for node, attrs in gx.node.items():
+        if (('python' not in (attrs.get('meta_yaml', {}).get('requirements', {}).get('host', []) or []))
```

Comment:
Can you store `attrs.get('meta_yaml', {}).get('requirements', {})` in a variable? It's hard to read here.

Reply:
Sure